app/mailers/application_mailer.rb

```diff
@@ -2,7 +2,9 @@
 
 class ApplicationMailer < ActionMailer::Base
   include Roadie::Rails::Automatic
-  default :from => Proc.new { Setting[:email_reply_address] || "noreply@foreman.example.org" }
+  default :raise_delivery_errors => true, :delivery_method => Proc.new { Setting[:delivery_method] },
```

Comment:
I don't think `raise_delivery_errors` can be set here, it just seems to create a header:

    raise-delivery-errors: true

shouldn't it be in the environment config?